docker/jenkins/Dockerfile.centos7-i386

```diff
@@ -111,6 +111,12 @@ RUN wget http://dl.fedoraproject.org/pub/epel/7/SRPMS/f/fakeroot-1.18.4-2.el7.sr
 # remove users/groups with colliding jenkins ids
 RUN userdel systemd-bus-proxy && userdel systemd-network && groupdel input
 
+# install Qt SDK
```

Comment:
you may want to consider moving the contents of this install file into the dockerfile, instead of copying the file in and executing it. it depends on how frequently you intend to change/use the script outside of the docker environment though.